### PR TITLE
Fixes per coding guidelines

### PR DESCRIPTION
**Pull request by idesai**
1. Removed camelCases
2. Applied code formatting
3. Other small fixes

### CODE REVIEW

**Comment by williamcroberts** [comment]
resolve conflicts...re-push.

**Comment by williamcroberts** [comment]
What code formatter did you use? Can we add it to CI to flunk patches that don't adhere?

**Comment by idesai** [comment]
> What code formatter did you use? Can we add it to CI to flunk patches that don't adhere?

Can we? I am not sure. I am using the formatter from this project itself https://github.com/tpm2-software/tpm2-tools/blob/master/misc/formatters/tpm2.0-eclipse-cdt-formatter.xml

**Comment by williamcroberts** [comment]
> > What code formatter did you use? Can we add it to CI to flunk patches that don't adhere?
> 
> Can we? I am not sure. I am using the formatter from this project itself https://github.com/tpm2-software/tpm2-tools/blob/master/misc/formatters/tpm2.0-eclipse-cdt-formatter.xml

Oh I didn't realize you got eclipse running....

File: lib/tpm2_session.c [hunk 1]
```diff
@@ -243,9 +239,9 @@ tool_rc tpm2_session_restore(ESYS_CONTEXT *ctx, const char *path, bool is_final,
 
     FILE *f = fopen(dup_path, "rb");
     if (!f) {
-        LOG_ERR("Could not open path \"%s\", due to error: \"%s\"",
-                dup_path, strerror(errno));
         free(dup_path);
+        LOG_ERR("Could not open path \"%s\", due to error: \"%s\"", dup_path,
+                strerror(errno));
```
Comment (williamcroberts) [inline lib/tpm2_session.c]: When you re-based you got the order of this hunk wrong, the free must go after the LOG_ERR statement. Please go to that commit and edit it so it's correct.

Comment (idesai) [reply lib/tpm2_session.c]: Thanks!

**Pull request merged**